server/command_hooks.go

```diff
@@ -185,3 +192,13 @@ func (p *Plugin) uploadFileTo(fileName string, contents io.Reader, channelID str
 
 	return file, nil
 }
+
+func (p *Plugin) hasPermissionToExportChannel(userID string) bool {
+	conf := p.getConfiguration()
+	if conf.EnableAdminRestrictions {
+		if !(p.client.User.HasPermissionTo(userID, model.PermissionManageChannelRoles) || p.client.User.HasPermissionTo(userID, model.PermissionManageSystem)) {
```

Comment:
You need to check the `model.permissionManageChannelRoles` for the channel scope, not system scope: https://developers.mattermost.com/integrate/reference/server/server-reference/#API.HasPermissionToChannel

That should make that check work.

Reply:
Great thanks! I tested that and it fixes the issue so the code should be updated.